### ISSUE TITLE
Plotly: savefig() _use_remote not defined & ERROR: MethodError: no method matching _show(::IOStream, ::MIME{Symbol("image/png")}, ::Plots.Plot{P lots.PlotlyBackend})

### ISSUE THREAD

**Issue by Helveg** [JuliaPlots/Plots.jl#1916]
When I try to save a plot with the Plotly backend it throws an error and produces an empty image if I save it like this:

    savefig(plot(rand(5)), "test.png")

Throwing the error:

```
ERROR: MethodError: no method matching _show(::IOStream, ::MIME{Symbol("image/png")}, ::Plots.Plot{P
lots.PlotlyBackend})
Stacktrace:
 [1] show(::IOStream, ::MIME{Symbol("image/png")}, ::Plots.Plot{Plots.PlotlyBackend}) at C:\Users\Ro
bin\.julia\packages\Plots\UQI78\src\output.jl:200
 [2] png(::Plots.Plot{Plots.PlotlyBackend}, ::String) at C:\Users\Robin\.julia\packages\Plots\UQI78\
src\output.jl:8
 [3] savefig(::Plots.Plot{Plots.PlotlyBackend}, ::String) at C:\Users\Robin\.julia\packages\Plots\UQ
I78\src\output.jl:123
 [4] top-level scope at none:0
```

And if I want to save it as "test.html" it produces this error:

```
ERROR: UndefVarError: _use_remote not defined
Stacktrace:
 [1] html(::Plots.Plot{Plots.PlotlyBackend}, ::String) at C:\Users\Robin\.julia\packages\Plots\UQI78
\src\output.jl:58
 [2] savefig(::Plots.Plot{Plots.PlotlyBackend}, ::String) at C:\Users\Robin\.julia\packages\Plots\UQ
I78\src\output.jl:123
 [3] top-level scope at none:0
```

**Comment by versipellis** [comment]
I have the same issue.

**Comment by mkborregaard** [comment]
Closed in #1926
